packages/react/src/Button/__tests__/__snapshots__/Button.test.tsx.snap

```diff
@@ -871,6 +874,7 @@ exports[`Button respects the small size prop 1`] = `
 
 .c0:disabled [data-component=ButtonCounter] {
   color: inherit;
+  background-color: var(--button-default-bgColor-disabled,undefined);
```

Comment:
This seems weird, but noticing the rest of the snapshot has similar `undefined` 🤷🏻

Reply:
Oh.. that's not good. Investigating

Reply:
Actually, this looks like a normal thing that happens. I tested it in Storybook and the value is there. Its not great but we're getting rid of snaps soon anyways. Thanks for flagging!

Reply:
I made a very similar comment. Ignore me if this is expected behavior.